<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-4ILC6H1" failures="1" tests="1" name="NegativeTests" time="9.076" errors="0" timestamp="2023-06-10T17:11:42 PST">
  <testcase classname="com.herokuapp.internet.NegativeTests" name="invalidUsernameTest" time="9.045">
    <failure type="java.lang.AssertionError" message="The username you entered is invalid
Actual message:
Your username is invalid!
×
Expected Message:
Your username is invalid!
 expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: The username you entered is invalid
Actual message:
Your username is invalid!
×
Expected Message:
Your username is invalid!
 expected [false] but found [true]
at com.herokuapp.internet.NegativeTests.invalidUsernameTest(NegativeTests.java:50)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- invalidUsernameTest -->
  <testcase classname="com.herokuapp.internet.NegativeTests" name="invalidPasswordTest" time="0.0">
    <ignored/>
  </testcase> <!-- invalidPasswordTest -->
</testsuite> <!-- NegativeTests -->
